### PR TITLE
Change old database names to new ones

### PR DESCRIPTION
**Pull request by bartoszbetka**

### CODE REVIEW

File: kubernetes/config-maps/concent-api-worker/local_settings.py.j2 [hunk 1]
```diff
@@ -13,11 +13,19 @@ from ..secrets.secrets import (
...
-DATABASES['default']['HOST']     = '{{ database_host }}'
-DATABASES['default']['PORT']     = '5432'
+DATABASES['control']['NAME']     = '{{ database_name }}'
+DATABASES['control']['USER']     = '{{ database_user }}'
+DATABASES['control']['PASSWORD'] = CONTROL_DATABASE_PASSWORD
+DATABASES['control']['HOST']     = '{{ database_host }}'
+DATABASES['control']['PORT']     = '5432'
+
+# Make sure that default database is empty
+DATABASES['default'] = {}
```
[the remark this answers is not in the document]
Comment (bartoszbetka) [reply kubernetes/config-maps/concent-api-worker/local_settings.py.j2]: The comment above explains why. We use it in all settings.

[the remark this answers is not in the document]
Comment (cameel) [reply kubernetes/config-maps/concent-api-worker/local_settings.py.j2]: But why isn't it already empty?

File: kubernetes/config-maps/concent-api-worker/local_settings.py.j2 [hunk 2]
```diff
@@ -13,11 +13,19 @@ from ..secrets.secrets import (
...
-DATABASES['default']['NAME']     = '{{ database_name }}'
-DATABASES['default']['USER']     = '{{ database_user }}'
-DATABASES['default']['PASSWORD'] = CONTROL_DATABASE_PASSWORD
-DATABASES['default']['HOST']     = '{{ database_host }}'
-DATABASES['default']['PORT']     = '5432'
+DATABASES['control']['NAME']     = '{{ database_name }}'
+DATABASES['control']['USER']     = '{{ database_user }}'
+DATABASES['control']['PASSWORD'] = CONTROL_DATABASE_PASSWORD
+DATABASES['control']['HOST']     = '{{ database_host }}'
+DATABASES['control']['PORT']     = '5432'
```
[the remark this answers is not in the document]
Comment (bartoszbetka) [reply kubernetes/config-maps/concent-api-worker/local_settings.py.j2]: Done.

File: kubernetes/config-maps/concent-api-worker/local_settings.py.j2 (hunk 1 above)
[the remark this answers is not in the document]
Comment (bartoszbetka) [reply kubernetes/config-maps/concent-api-worker/local_settings.py.j2]: I don't remember if this setting is still needed after `default` database name was set for empty in `concent` side. But thanks to this setting we are sure that the database is empty.

[the remark this answers is not in the document]
Comment (bartoszbetka) [reply kubernetes/config-maps/concent-api-worker/local_settings.py.j2]: Ok, I add assertion in this place.

File: kubernetes/config-maps/concent-api-worker/local_settings.py.j2 [hunk 3]
```diff
@@ -13,11 +13,19 @@ from ..secrets.secrets import (
...
+
+# Make sure that default database is empty
+assert DATABASES['default'] == {}
```
Comment (cameel) [inline kubernetes/config-maps/concent-api-worker/local_settings.py.j2]: You should explain why you expect it to be empty.

```
# The `default` database is not used by the application but Django still requires the key to be present
# in `DATABASES` dict. Make sure it's really empty (if it's not it must be some kind of error).
```

**Pull request merged**